README.md

```diff
@@ -39,6 +39,7 @@ UI by [@DennisBednarz](https://twitter.com/DennisBednarz) & [Samg_is_a_Ninja](ht
 * Button to restart device
 * Button to open Cydia in case it doesn't appear on the Home Screen
 * Label to show the days left till the application expires
+* Buitl-In Update checker
```

Comment:
*Built-In

Reply:
I see... I am editing on iPad so doing a lot of mistakes.. sorry 😇

Reply:
Resolved..thanks